client/src/pages/Register/secondStep.jsx

```diff
@@ -31,8 +35,12 @@ const SecondStepSignup = ({ handleSubmit, values, setValues }) => {
 						id="register-type-input"
 						value={type}
 						onChange={({ target: { value } }) => setValues({ type: value, aboutme, address }) }
-						options={personTypes}
-						label={t.whoAmI} />
+						options={dropdownOptions}
+						label={t.whoAmI}
+						helperText={typeError ?? undefined}
+						error={typeError !== null}
+						onFocus={inputFocus}
```

Comment:
the onFocus and onBlur are used to manage the dirtiness of the input, just as in step 1